tests/console/gpg.pm

```diff
@@ -0,0 +1,65 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: gpg key generation and passphrase test
+# Maintainer: Petr Cervinka <pcervinka@suse.com>
+
+use base "consoletest";
+use strict;
+use testapi;
+
+sub run() {
+    select_console 'root-console';
```

Comment:
I will change it to user-console and re-run tests in private openQA instance and provide new links when ready.